community/kernel/src/main/java/org/neo4j/kernel/impl/api/KernelTransactions.java

```diff
@@ -135,22 +135,26 @@
     private volatile boolean stopped = true;
 
     public KernelTransactions( StatementLocksFactory statementLocksFactory,
-            ConstraintIndexCreator constraintIndexCreator, StatementOperationParts statementOperations,
-            SchemaWriteGuard schemaWriteGuard, TransactionHeaderInformationFactory txHeaderFactory,
-            TransactionCommitProcess transactionCommitProcess, IndexConfigStore indexConfigStore,
-            ExplicitIndexProviderLookup explicitIndexProviderLookup, TransactionHooks hooks,
-            TransactionMonitor transactionMonitor, AvailabilityGuard availabilityGuard, Tracers tracers,
-            StorageEngine storageEngine, Procedures procedures, TransactionIdStore transactionIdStore,
-            SystemNanoClock clock,
-            AtomicReference<CpuClock> cpuClockRef, AtomicReference<HeapAllocation> heapAllocationRef, AccessCapability accessCapability,
-            Supplier<DefaultCursors> cursorsSupplier,
-            AutoIndexing autoIndexing,
-            ExplicitIndexStore explicitIndexStore,
-            VersionContextSupplier versionContextSupplier,
-            CollectionsFactorySupplier collectionsFactorySupplier,
-            ConstraintSemantics constraintSemantics,
-            SchemaState schemaState,
-            IndexingService indexingService )
+                               ConstraintIndexCreator constraintIndexCreator,
+                               StatementOperationParts statementOperations,
+                               SchemaWriteGuard schemaWriteGuard, TransactionHeaderInformationFactory txHeaderFactory,
+                               TransactionCommitProcess transactionCommitProcess, IndexConfigStore indexConfigStore,
+                               ExplicitIndexProviderLookup explicitIndexProviderLookup, TransactionHooks hooks,
+                               TransactionMonitor transactionMonitor, AvailabilityGuard availabilityGuard,
+                               Tracers tracers,
+                               StorageEngine storageEngine, Procedures procedures,
+                               TransactionIdStore transactionIdStore,
+                               SystemNanoClock clock,
+                               AtomicReference<CpuClock> cpuClockRef, AtomicReference<HeapAllocation> heapAllocationRef,
+                               AccessCapability accessCapability,
+                               Supplier<DefaultCursors> cursorsSupplier,
+                               AutoIndexing autoIndexing,
+                               ExplicitIndexStore explicitIndexStore,
+                               VersionContextSupplier versionContextSupplier,
+                               CollectionsFactorySupplier collectionsFactorySupplier,
+                               ConstraintSemantics constraintSemantics,
+                               SchemaState schemaState,
+                               IndexingProvidersService indexProviders )
```

Comment:
In general, we don't use this kind of parameter alignment. Let's try not to flip back and forth by auto formatting.